docs/milestone2/milestone2_application.yaml

```diff
@@ -1,39 +1,39 @@
 apiVersion: v1
```

Comment:
These milestone 2 files can be removed on main and live on on the milestone 2 tag.
The steps won't work on main regardless of the mctc/mgc change. (relies on syncer pre-ocm changes)

I'll PR that separate after merge of this if it's simpler

Reply:
There seems to be quite a few things that need to be removed from this repo, syncer, RLP controller, outdated docs etc ... I'd rather just merge this, and do other follow up PRs for the rest if possible.

Reply:
Follow up to remove RLP and milestone 2 docs https://github.com/Kuadrant/multicluster-gateway-controller/pull/223